ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/ContinuousHikingStateMachine/StopContinuousHikingTransitionCondition.java

```diff
@@ -1,17 +1,18 @@
-package us.ihmc.behaviors.activeMapping;
+package us.ihmc.behaviors.activeMapping.ContinuousHikingStateMachine;
 
 import behavior_msgs.msg.dds.ContinuousWalkingCommandMessage;
+import us.ihmc.behaviors.activeMapping.ContinuousHikingParameters;
 import us.ihmc.robotics.stateMachine.core.StateTransitionCondition;
 
 import java.util.concurrent.atomic.AtomicReference;
 
-public class StopContinuousWalkingTransitionCondition implements StateTransitionCondition
+public class StopContinuousHikingTransitionCondition implements StateTransitionCondition
 {
    private final ContinuousHikingParameters continuousHikingParameters;
    private final AtomicReference<ContinuousWalkingCommandMessage> commandMessage;
 
-   public StopContinuousWalkingTransitionCondition(ContinuousHikingParameters continuousHikingParameters,
-                                                   AtomicReference<ContinuousWalkingCommandMessage> commandMessage)
+   public StopContinuousHikingTransitionCondition(ContinuousHikingParameters continuousHikingParameters,
+                                                  AtomicReference<ContinuousWalkingCommandMessage> commandMessage)
```

Comment:
From a consistency perspective, the ordering should be thes ame here as in StartContinuous....

Reply:
Good catch